device/demo/rooms/test/test.tscn

```diff
@@ -12,10 +12,12 @@
 [ext_resource path="res://demo/rooms/test/ice_cream.tscn" type="PackedScene" id=10]
 [ext_resource path="res://demo/rooms/test/box.tscn" type="PackedScene" id=11]
 [ext_resource path="res://demo/rooms/test/can.tscn" type="PackedScene" id=12]
-[ext_resource path="res://demo/rooms/test/chain_saw.tscn" type="PackedScene" id=13]
-[ext_resource path="res://demo/rooms/test/floppy.tscn" type="PackedScene" id=14]
-[ext_resource path="res://globals/game.tscn" type="PackedScene" id=15]
-[ext_resource path="res://demo/ui/rmb_hints.gd" type="Script" id=16]
+[ext_resource path="res://demo/rooms/test/can_anims.gd" type="Script" id=13]
```

Comment:
This file is missing from the PR.

Reply:
I created the following file for testing, based on your animations, which I think should look similar to your file:

    # angle "0" is facing the camera, rotates counter clock-wise, angle 90 is profile right, etc

    const dir_angles = [ 90, 180, 270, 360 ]

    const directions = ["face_right", 1, "face_up", 1, "face_left", 1, "face_down", 1]

    const idles = ["face_right", 1, "face_up", 1, "face_left", 1, "face_down", 1]

    const speaks = ["face_right", 1, "face_up", 1, "face_left", 1, "face_down", 1]

`idles` and `speaks` is probably not necessary though, but I haven't looked into that.

Reply:
The file is force-pushed, though you pretty much got it.